tests/coverage/Future.md

```diff
@@ -0,0 +1,15 @@
+# Ideas for future testing
```

Comment:
This should not be put here. Let's open an Issue or a Discussion about it instead.

Reply:
Sounds reasonable—it's like a roadmap for testing, if someone likes to pick up on any of these items.
(No idea if the list is self-explanatory though.)

The main readme could link to this Discussion as well as to the Website (above; on Approaches).